### ISSUE TITLE
[1.20.1] Add Mekanism-Support back

### ISSUE THREAD

**Issue by Robocraft999** [Robocraft999/CreateGoggles#24]
Mekanism is updated and the support could be readded

**Comment by batmanrobin68002** [comment]
yes pls. cause i loved that unit. and with the goggles curio. it looks terrible when shown so it would be nice for mekinism support to return that way i can show the goggles without it looking dumb
